frontend/src/services/OpeningService.js

```diff
@@ -39,3 +39,89 @@ export async function fetchRecentOpenings() {
         throw error;
     }
 }
+
+export async function fetchOpeningsPerYear(orgUnitCode, statusCode, entryDateStart, entryDateEnd) {
+    let authToken = await getAuthIdToken();
+    try {
+        // Construct URL with optional parameters
+        let url = 'https://nr-silva-test-backend.apps.silver.devops.gov.bc.ca/api/dashboard-metrics/submission-trends';
```

Comment:
Two blocks:
 - why is this hard coded?
 - why is it using TEST?

Reply:
It is hardcoded, coz for now we are just relying on the test data for the api endpoints. We are not ready for PROD yet, once the endpoints are well tested we will change the url so it comes form the env variables rather than always form test.

Reply:
So that means you don't want this corrected and aren't ready to merge? If so, please switch to draft status. @jazzgrewal

Reply:
SILVA app in general is not in PROD. Not a single user is using the prod link as of yet. If this is a emergency fix for you, I can change it to env variable. At least it saves us the time spent on arguments lol.

Reply:
@jazzgrewal Want me to tackle that?  It's not a big deal.